### DISCUSSION TITLE
[FEATURE REQUEST] Emmet for MD files.

### DISCUSSION THREAD

**Discussion by L1ghtingBolt** [withastro/roadmap#110] [Proposal]
It would be good if the Astro VSCode extension would provide Emmet snippets for MarkDown files.  Also, Tailwind does not fully support the MD files, only some of te classes work.
EDIT: Tailwind does support MD, I just forgot to add MD files in TW configuration file.
EDIT 2: A workaround for Emmet is installing MDX language support and writing this on JSON VSCode Config:
"emmet.includeLanguages": {
        "mdx": "html"
    },
    "files.associations": {
        "*.md": "mdx"
    }

**Comment by natemoo-re** [comment]
Would you mind opening this as an issue on https://github.com/withastro/language-tools? I think that would be a better forum for this request!
